### ISSUE TITLE
Charts without accompanying text should always show in left column

### ISSUE THREAD

**Issue by HannahRitchie** [owid/owid-grapher#1112]
## Core problem

It’s currently the case that if a chart is on its own (without text), it appears in the right column. Example [here](https://ourworldindata.org/food-prices#recent-trends-in-food-price-volatility). This looks a bit broken. Authors can manually put it in the left column, but this is quite a lot of work.

## Proposed solution

We would propose to set two defaults:

- If a single chart appears below a heading without text, it always shows in the left column
- If there are two charts, they automatically show side-by-side in two-column view.

**Comment by mlbrgl** [comment]
> If there are two charts, they automatically show side-by-side in two-column view.

This is [already happening](https://ourworldindata.org/food-prices#food-consumed-away-from-home), unless I'm misunderstanding something @JoeHasell ?

> If a single chart appears below a heading without text, it always shows in the left column

I'm rephrasing this as: "A single chart (without companion content in the next column) always shows in the left column. Authors cannot override this (it is considered a layout error)." 
Please let me know if that makes sense to you, thanks!

**Comment by JoeHasell** [comment]
> I'm rephrasing this as: "A single chart (without companion content in the next column) always shows in the left column. Authors cannot override this (it is considered a layout error)."

Yes that makes sense to me. I can't imaging a scenario in which an override would be helpful.

> This is [already happening](https://ourworldindata.org/food-prices#food-consumed-away-from-home), unless I'm misunderstanding something @JoeHasell ?

My guess is that this was just to disambiguate – i.e. that it shouldn't be two charts stacked on top of each other in the left column.

But incidentally the example [you link to](https://ourworldindata.org/food-prices#food-consumed-away-from-home) is also not great, because of the uneven columns. I don't think that's such a common case, and almost certainly a lower priority than

**Comment by mlbrgl** [comment]
Updated rules after call with @JoeHasell:
- 1 standalone chart alone in right column -> switch columns to sticky-left layout (left column bigger than right) and move chart to left column.
- 2 standalone charts: unchanged (`side-by-side` layout already applied)
- 3+ standalone charts: ideally, fill up `side-by-side` columns from left to right to preserve alignment. Fallback if too complex: `side-by-side` layout for pairs and `sticky-left` for remaining chart (effectively combining the two previous scenarios).

This logic will be applied to grapher charts (and possibly images with a non-h6 caption).

> I'm rephrasing this as: "A single chart (without companion content in the next column) always shows in the left column. ~~Authors cannot override this (it is considered a layout error).~~"

Change of hearts: creating a "wrong" layout cannot really happen by accident; the path of least resistance leading to create a correct layout by default. Preserving overrides makes reasoning with the system simpler.
